code/modules/SCP/SCPs/SCP-173.dm

```diff
@@ -334,142 +320,145 @@ GLOBAL_LIST_EMPTY(scp173s)
-		playsound(loc, 'sound/machines/airlock_creaking.ogg', 50, 1)
-		if(damage_state < damage_max)
-			visible_message(SPAN_WARNING("You hear the sound of metal [pick("groaning","shearing","tearing","bending")]!"))
-			damage_state += 1
-			updateIconandDesc()
-		else
-			visible_message(SPAN_DANGER("[current173] bends open \the [src]!"))
-			damage_state = damage_max + 1
-			ReleaseContents()
+	visible_message(SPAN_WARNING("[L] attempts to open \the [src]."))
+	if(do_after(L, 5 SECONDS, src))
+		visible_message(SPAN_DANGER("[L] opens \the [src]!"))
+		ReleaseContents()
+
+/obj/structure/scp173_cage/relaymove(mob/living/scp_173/user, direction)
+	if(resist_cooldown > world.time)
+		return
+	if(user.IsBeingWatched())
+		return
+	resist_cooldown = world.time + 5 SECONDS
+	if(!do_after(user, 5 SECONDS, src))
+		return
+	if(user.IsBeingWatched())
+		to_chat(user, SPAN_WARNING("Someone is looking at you!"))
+		return
+	damage_state += 1
+	update_icon()
+	if(damage_state < damage_state_max)
+		visible_message(SPAN_WARNING("[user] damages \the [src]!"))
+		playsound(src, 'sound/effects/grillehit.ogg', 35, TRUE)
+		return
+	visible_message(SPAN_DANGER("[user] opens \the [src] from the inside!"))
+	ReleaseContents()
 
-/obj/structure/scp173_cage/proc/updateIconandDesc() //Updates cage icon and description according to current damage state and contents
+/obj/structure/scp173_cage/examine(mob/user, distance, infix = "", suffix = "")
+	. = ..()
+	for(var/mob/M in contents)
+		to_chat(user, "[icon2html(M, user)] It has [M] inside of it!")
+	switch(damage_state)
+		if(1, 2)
+			to_chat(user, SPAN_NOTICE("It looks slightly damaged."))
+		if(3, 4)
+			to_chat(user, SPAN_WARNING("It is seriously damaged!"))
+		if(5 to INFINITY)
+			to_chat(user, SPAN_DANGER("It is completely broken!"))
+
+// Updates icon state and underlays
+/obj/structure/scp173_cage/update_icon()
 	underlays.Cut()
 
-	if((damage_state <= damage_max) && ((!LAZYLEN(contents)) || !current173))
+	if(!LAZYLEN(contents))
+		plane = initial(plane)
 		icon_state = "open"
-		desc = initial(desc)
-	else if(damage_state == 0)
-		icon_state = "closed"
-		desc = "A cage for containing SCP-173. It is currently holding [current173]."
-	else if(damage_state <= 1)
-		icon_state = "damage_1"
-		desc = "A cage for containing SCP-173. It is currently holding [current173]. It looks slightly damaged."
-	else if((damage_state >= Clamp((damage_max - 1), 2, INFINITY)) && (damage_state <= damage_max)) //Dont want the damage visuals getting fucky because damage_max was too low
-		icon_state = "damage_2"
-		desc = "A cage for containing SCP-173. It is currently holding [current173]. It looks fairly damaged."
-	else if(damage_state > damage_max) //Damage state is equal to damage max last stage before breaking, so this is neccesary
-		icon_state = "damage_3"
-		desc = "An empty cage for containing SCP-173. It appears broken."
-
-	if((LAZYLEN(contents)) && current173)
-		underlays += image(current173)
+		return
```

Comment:
This causes the cage sprite icon_state to remain "open" even when the cage is broken, as 173 is no longer inside and  if(!LAZYLEN(contents)) returns true.

Reply:
you can ignore this though I'm sure youre aware of it